### DISCUSSION TITLE
2024-01-11 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#2756] [Community Calls]
The next Modules Feedback Session will be on **Thursday, Jan 11th, 2024 at 1600 UTC.**

See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

Please reply with any questions/topics you would like to see covered.

### Rough agenda

* `cue get go` and modules - how do they play together?
* Use cases that currently "require" `cue get go` and modules - do we still need/want `cue get go` in all situations? e.g. using k8s types

### Meeting details
https://meet.google.com/wis-hrtw-isf

This session will be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/AdKY-dqlFC8

### Slides/materials
https://docs.google.com/presentation/d/1RFlG8JWqmbUXqPABtFQt7yNK9WD1BU7orG7EPFno3co

### Auto-generated transcript
https://docs.google.com/document/d/1HgV83NUzxnZfMNXUuI6RybKTP57q-lXk3rtkwwBSEIY/edit?usp=sharing

**Comment by cedricgc** [comment]
Curious any thought of the interplay of modules and code generation. For example, would it make sense to generate and vendor K8s definitions for every app or to create 'canonical' modules for using K8s?

**Reply by myitcv** [reply]
@cedricgc thanks for brining this up. The topic of `cue get go` (and the slightly broader use cases that currently require `cue get go`) will be the focus of this session. See also #2746

**Comment by myitcv** [comment]
I've updated the description above with a brief two-point agenda. We plan to focus much of the discussion (beyond updates) on how generated code (`cue get go` or otherwise) and use cases requiring generated code will interact with modules. We would very much welcome and encourage folks who use `cue get go` to join the call and contribute to discussion.

**Comment by dominikdm** [comment]
Added recording, slides, and transcript.
